tests/sles4sap/hana_install.pm

```diff
@@ -89,7 +89,13 @@ sub run {
         $hdblcm = "/sapinst/DATA_UNITS/HDB_SERVER_LINUX_PPC64/hdblcm";
         $ret    = script_run "ls $hdblcm";
     }
-    die "hdblcm is not in $hdblcm" unless (defined $ret and $ret == 0);
+    # hdblcm path can be different depending on the type of installation master in use
+    if ($ret) {
```

Comment:
Shouldn't this condition be `if (!ret)`?. Otherwise if it was found earlier, why check once again?.

Reply:
Basically because older versions of Hana for ppc64le have the installation binary in `/sapinst/DATA_UNITS/HDB_SERVER_LINUX_PPC64/hdblcm` instead of `/sapinst/DATA_UNITS/HDB_SERVER_LINUX_PPC64LE/hdblcm`, so the previous `if` block is only correcting that path for ppc64le. The second `if` block deals with the `hdblcm` command not being present in either location, which are the paths where `hdblcm` is in the full installation masters that are available for download in sap.com. Then, if it's not found there, I'm assuming that it's in `/sapinst/SAP_HANA_DATABASE/hdblcm` which is the path in the packages for the update/patch level versions of Hana